app/templates/index.html

```diff
@@ -8,6 +8,9 @@
             <p>{{ todo.task }}</p>
             <br>
             {{ todo.description }}
+            <form action="{{ url_for('main.delete_todo', todo_id=todo.id) }}" method="POST" style="display:inline;">
+                <button type="submit" class="btn btn-danger">Delete</button>
```

Comment:
button should be be more of outline style and not red

Reply:
I think red is better since it hints risk in role of the button to delete

Reply:
cool, keep it like that but update the style in the mock up